rust/config/mainnet2/mainnet2_config.json

```diff
@@ -152,6 +152,25 @@
       "index": {
         "from": "2595747"
       }
+    },
+    "gnosis": {
+      "name": "gnosis",
+      "domain": "100",
+      "addresses": {
+        "mailbox": "0x35231d4c2D8B8ADcB5617A638A0c4548684c7C70",
+        "interchainGasPaymaster": "0x6cA0B6D22da47f091B7613223cD4BB03a2d77918",
+        "validatorAnnounce": "0xB5bc6369bc2Dc8969B18f6C25488EC0a38cA637f"
+      },
+      "signer": null,
+      "protocol": "ethereum",
+      "finalityBlocks": "3",
```

Comment:
How was this determined? Wondering why it's not the same as ethereum given they run the same consensus protocol

Reply:
I went through the block explorer and the past year of reorgs which never exceeded 1. I moved it up to 14 which is what coinbase is using on eth for now

Reply:
Yeah I mean tbh the value we're using for eth seems very conservative. But i'd prefer to be consistent

Reply:
I'll move them all up to 14 then